### PR TITLE
update driver dependencies

### PR DESCRIPTION
**Pull request by nvollmar**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -21,15 +21,15 @@ object Dependencies {
...
+  val driverVersion = "4.17.0"
+  val driverVersionInDocs = "4.17"
 
   val pekkoConnectorsVersion = "1.0.1"
   val pekkoConnectorsVersionInDocs = "1.0"
   // for example
   val pekkoManagementVersion = "1.0.0"
 
-  val logback = "ch.qos.logback" % "logback-classic" % "1.2.13"
+  val logback = "ch.qos.logback" % "logback-classic" % "1.3.14"
```
Comment (pjfanning) [inline project/Dependencies.scala:32]: do we need to upgrade logback?

Comment (nvollmar) [reply project/Dependencies.scala:32]: no, just aligned the versions with what Pekko Core is using: https://github.com/apache/incubator-pekko/blob/959c98dba58537ac226971efaef576d0ac0fdbbb/project/Dependencies.scala#L34C13-L34C13

Comment (nvollmar) [reply project/Dependencies.scala:32]: right, I'll revert that

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**